framework/lib/components/pin-input/types.ts

```diff
@@ -1,6 +1,6 @@
 import { PinInputProps as ChakraPinInputProps } from '@chakra-ui/react'
 
-export type PinVariant = 'green' | 'yellow' | 'gray' | 'red'
+export type PinVariant = 'running' | 'inProgress' | 'notExecuted' | 'rejected'
 export type PinSize = 'sm' | 'md' | 'lg'
```

Comment:
Why have the names of the variants been changed? this would be a breaking change on multiple places in domains. If we choose to go with variable names then please make the PR as a breaking change and make sure that it it bumped and fixed in those places where the old variant names are used.

Reply:
It is also possible to keep the old and add the new at the same time. 
That way, it is not a breaking change anymore and we can migrate without a major new version.

Reply:
I have added both old and new variant names